### PR TITLE
Add timestamp to deploy proto

### PR DESCRIPTION
**Pull request by birchmd**
## Overview
Gives some uniquely identifying information to a deploy for use in creating deterministic unforegable names.

### Does this PR relate to an RChain JIRA issue? 
https://rchain.atlassian.net/browse/RHOL-402

### Complete this checklist before you submit the PR
- [x] This PR contains no more than 200 lines of code, excluding test code.
- [x] This PR meets [RChain development coding standards](https://rchain.atlassian.net/wiki/spaces/DOC/pages/28082177/Coding+Standards).
- [x] You have someone in mind to assign for review. Make this assignment after submitting the PR.

### Notes
Optional. Add any notes on caveats, approaches you tried that didn't work, or anything else.

### CODE REVIEW

File: models/src/main/protobuf/CasperMessage.proto [hunk 1]
```diff
@@ -20,10 +20,10 @@ service DeployService {
 }
 
 message DeployString {
-  bytes  user  = 1; //public key
-  int32  nonce = 2; //for uniqueness
-  string term  = 3; //rholang source code to deploy (will be parsed into `Par`)
-  bytes  sig   = 4; //signature of (hash(resource) + nonce) using private key
+  bytes  user      = 1; //public key
```
Comment (KentShikama) [inline models/src/main/protobuf/CasperMessage.proto]: Can we add a // TODO: Potentially remove if we are using ECDSA

File: casper/src/main/scala/coop/rchain/casper/util/comm/DeployRuntime.scala [hunk 2]
```diff
@@ -38,10 +38,9 @@ object DeployRuntime {
     Try(Source.fromFile(file).mkString) match {
       case Success(code) =>
         for {
-          //TODO: have the client track the nonce
-          nonce <- Capture[F].capture { scala.util.Random.nextInt(10000) }
+          timestamp <- Capture[F].capture { System.nanoTime() }
```
Comment (KentShikama) [inline casper/src/main/scala/coop/rchain/casper/util/comm/DeployRuntime.scala]: Divide by 1000?

File: models/src/main/protobuf/CasperMessage.proto (hunk 1 above)
Comment (birchmd) [reply models/src/main/protobuf/CasperMessage.proto]: I personally don't mind there being some redundant information in messages. Redundancy gives you something to verify, a way to check the message has not been corrupted. I also think giving freedom in the signature algorithm, as we do with blocks is a good idea.

**Comment by birchmd** [comment]
@KentShikama Anything else you want addressed?

**Pull request merged**